js/templates/account.html

```diff
@@ -12,6 +12,9 @@ <h2 class="mail-account-email" title="{{email}}">{{email}}</h2>
 
 <div class="app-navigation-entry-menu">
 	<ul>
+		<li><button class="icon-rename svg" title="settings"></button></li>
+	</ul>
+	<ul>
 		<li><button class="icon-delete svg" title="delete"></button></li>
```

Comment:
please use ``{{ t 'Delete account' }}`` for L10n (I forgot that in the account popover menu PR)